File: src/main/java/me/testaccount666/serversystem/exceptions/NotDirectoryException.java
package me.testaccount666.serversystem.exceptions;

import java.io.IOException;
import java.io.Serial;

public class NotDirectoryException extends IOException {

    @Serial
    private static final long serialVersionUID = 2917557613235459376L;

    public NotDirectoryException(String message) {
        super(message);
    }

    public NotDirectoryException(String message, Throwable cause) {
        super(message, cause);
    }

    public NotDirectoryException(Throwable cause) {
        super(cause);
    }

    public NotDirectoryException() {
    }
}

File: src/main/kotlin/me/testaccount666/serversystem/exceptions/NotDirectoryException.kt
package me.testaccount666.serversystem.exceptions

import java.io.IOException
import java.io.Serial

class NotDirectoryException : IOException {
    constructor(message: String?) : super(message)

    constructor(message: String?, cause: Throwable?) : super(message, cause)

    constructor(cause: Throwable?) : super(cause)

    constructor()

    companion object {
        @Serial
        private const val serialVersionUID = 2917557613235459376L
    }
}

